docs/pages/dictionary.mdx

```diff
@@ -0,0 +1,262 @@
+import { Callout } from "nextra/components";
+
+# Solidity to MUD Dictionary
+
+To properly use MUD you need to do some things differently than the way you do them normally in Solidity.
+
+## State variables
+
+A `System` should be stateless, for several reasons:
```

Comment:
```suggestion
A `System` contract should not store any internal state, but only interact with `World` state via `Tables`.
```
Making it more explicit that systems can work with state, just not internal state